app/src/main/java/org/mozilla/tv/firefox/FirefoxApplication.kt

```diff
@@ -55,6 +56,11 @@ open class FirefoxApplication : LocaleAwareApplication() {
         }
     }
 
+    private fun initServiceLocator() {
+        val getLanguage = { LocaleManager.getInstance().getCurrentLanguage(this) }
```

Comment:
Can this be declared inside `ServiceLocator` (which already has access to `applicationContext`) or even `ViewModelFactory`? It's an unnecessary dependency for the factory class when it's only used for `PocketViewModel`

Reply:
Good point.  I'll pass application into the factory and grab this in there.